src/components/templates/ScreeningRoom/ScreeningRoom.tsx

```diff
@@ -120,7 +120,12 @@ export const ScreeningRoom: React.FC<ScreeningRoomProps> = ({ venue }) => {
   );
 
   return (
-    <div className="ScreeningRoom">
+    <div
+      data-bem="ScreeningRoom"
+      data-block="ScreeningRoom"
+      data-side="att"
+      className="ScreeningRoom"
```

Comment:
You're right about the `className` I just didn't want to make a blind assumption and didn't want to spend too much time on checking every instance of it.

As the large number, yes, `data-bem`  would usually be the only one we need, however, `data-bem="Input"` can be added in two components, one attendee side and another admin side, hence `data-side`. The `data-block` on the other hand should be just one, the top level of those building blocks according to the visual redesign to make it easy to find them all like `document.querySelectorAll('[data-block')`

Reply:
ok, I got `data-side` usage, although I couldn't understand what "att" means. But still why do we need `data-block`? as it seems like the copy of `data-bem`.

I don't want us to become obsessed on all the `data-*` attributes now, and then just find out that we ignore them and realize some day that's it's again tons of legacy not-needed code. 
do you often use `document.querySelector` instead of just instecting elements on the html tab?

to me, it seems that data-bem should be pretty enoough.